### PR TITLE
980 ams election samlow

### LINKED ISSUE #980: AMS Template
Create a template for AMS articles for 2023. Referendum menu to be above div containing feature image. Template available under select stock layout in wagtail. Idea: have the menu be derived from any h2 with a id created in the article.

### PR DESCRIPTION
**Pull request by SamuelmdLow**
## What problem does this PR solve?

fixed issue #980 

## How did you fix the problem?

Created a page links block and a header block. Clicking a page link will send the user to the header which shares the id

### CODE REVIEW

**Review by FireBoyAJ24 — CHANGES_REQUESTED** [review]
Can we add all the inline styles and the scripts to be in external style sheets and scripts?

File: article/templates/article/stream_blocks/pageLink.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% load static %}
+
+<div class="header-menu">
+     <ul>
+          {% for item in self.list %}
+               <style>
+                    .pageLink{{item.id}} {
+                         background-color: #{{item.colour}} !important;
```
Comment (FireBoyAJ24) [inline article/templates/article/stream_blocks/pageLink.html]: Is there anyway to avoid !important?

**Review by FireBoyAJ24 — APPROVED** [review]
The !important is a small aspect and will hopefully not impact the rest of the code base but it is best avoided unless all options have been dried up.

File: article/templates/article/stream_blocks/pageLink.html (hunk 1 above)
Comment (SamuelmdLow) [reply article/templates/article/stream_blocks/pageLink.html]: I've removed it. Turns out it wasn't actually important

**Comment by SamuelmdLow** [comment]
Merged into redesign

**Pull request closed without merging**